examples/hybrid_jobs/1_Quantum_machine_learning_in_Amazon_Braket_Hybrid_Jobs/Quantum_machine_learning_in_Amazon_Braket_Hybrid_Jobs.ipynb

```diff
@@ -62,11 +62,11 @@
      "text": [
```

Comment:
<p>Why does <code>n_layers=1</code> stop at 6? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/458/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>I believe it just converged early. We only pass in an option called "maxiter" to the minimizer. The job succeeded and combing through the cloudwatch logs, I only see the six metrics logged.</p>